wpilibc/sim/include/NetworkCommunication/FRCComm.h

```diff
@@ -47,13 +47,13 @@ enum MatchType_t {
 };
 
 struct ControlWord_t {
-  uint32_t enabled : 1;
-  uint32_t autonomous : 1;
-  uint32_t test : 1;
-  uint32_t eStop : 1;
-  uint32_t fmsAttached : 1;
-  uint32_t dsAttached : 1;
-  uint32_t control_reserved : 26;
+  int enabled : 1;
+  int autonomous : 1;
+  int test : 1;
+  int eStop : 1;
+  int fmsAttached : 1;
+  int dsAttached : 1;
+  int control_reserved : 26;
```

Comment:
I don't think we want to make changes to this file--shouldn't this match the hardware one (structures and function signatures)?